Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m

```diff
@@ -194,7 +195,7 @@ - (void)createContents
     unsigned long long fileSize =
         [[NSFileManager defaultManager] attributesOfItemAtPath:[self.attachmentStream filePath] error:&error].fileSize;
     OWSAssert(!error);
-    NSString *bottomText = [ViewControllerUtils formatFileSize:fileSize];
+    NSString *bottomText = [OWSFormat formatFileSize:fileSize];
```

Comment:
I pulled out the formatting functionality (which we want to share with SAE) from`ViewControllerUtils` (which has other functionality we don't want to share) into `OWSFormat`.